scripts/downgrade.sh.in

```diff
@@ -124,7 +124,7 @@ launch_downgrade () {
     "${SALT_MASTER_CALL[@]}" salt-run state.orchestrate \
         metalk8s.orchestrate.downgrade \
         saltenv="$SALTENV" \
-        "${FORCE:+pillar={'metalk8s': {'downgrade': {'enabled': true\}\}\}}"
+        ${FORCE:+pillar="{'metalk8s': {'downgrade': {'enabled': true\}\}\}"}
```

Comment:
We must not escape the ending curly braces if we escape the string within the variable expansion:
```suggestion
        ${FORCE:+pillar="{'metalk8s': {'downgrade': {'enabled': true}}}"}
```
Otherwise we will end up with an invalid dict.

Reply:
Right :+1: